.projenrc.js

```diff
@@ -21,29 +21,39 @@ const project = new JsiiProject({
     "jsii"
   ],
 
-  java: {
```

Comment:
Currently only releasing to npm because jsii does not have support for pre-releases
@nija-at @RomainMuller @MrArnoldPalmer any estimate when this will be possible? Do we have a tracking issue?

Reply:
No tracking issue yet. Tracking currently in the project - https://github.com/aws/aws-cdk/projects/10#card-44000307. 

I expect this work to be picked up and completed this month.